scripts/typescript_checks.py

```diff
@@ -74,7 +77,12 @@ def compile_and_check_typescript(config_path):
     if os.path.exists(COMPILED_JS_DIR):
         shutil.rmtree(COMPILED_JS_DIR)
 
+    # The value of process.stdout can be None but according to the
+    # arguments passed here, it should not be None.
+    # Hence add an assert.
```

Comment:
I find the wording here a little confusing. What about:

```
The value of `process.stdout` should not be None since we passed the `stdout=subprocess.PIPE` argument to `Popen`
```

This avoids the vague contrast between "can be None" and "should not be None" in the current comment

Reply:
done